crates/webidl/src/util.rs

```diff
@@ -285,3 +286,40 @@ pub fn create_setter(
         vec![backend::ast::BindgenAttr::Setter(Some(raw_ident(name)))],
     )
 }
+
+pub fn wrap_import_function(function: backend::ast::ImportFunction) -> backend::ast::Import {
+    backend::ast::Import {
+        module: None,
+        version: None,
+        js_namespace: None,
+        kind: backend::ast::ImportKind::Function(function),
+    }
+}
+
+/// ChromeOnly is for things that are only exposed to priveleged code in Firefox.
+pub fn is_chrome_only(ext_attrs: &[Box<ExtendedAttribute>]) -> bool {
+    ext_attrs.iter().any(|external_attribute| {
+        return match external_attribute {
```

Comment:
To remove the `box` pattern, replace `external_attribute` with `&**external_attribute`